CHANGELOG.md

```diff
@@ -33,8 +37,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - I18n Ca, Cs, Da, De, El, Fi, Fr, Ko, Nl, Pl, Ro, Ru, Sk, Sl, Sv and Uk.
 
-## [3.4.2] - 2021-08-27
-
```

Comment:
@sheilagomes I guess you moved this by mistake (now it's under the Arabic translation changelog). Could you move it back?

Reply:
Done! Sorry about that.